pkg/v1/tkg/tkgctl/client.go

```diff
@@ -144,6 +144,15 @@ func New(options Options) (TKGClient, error) { //nolint:gocritic
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to ensure prerequisites")
 	}
+	tkgConfigFile, err := allClients.TKGConfigPathsClient.GetTKGConfigPath()
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get TKG config file path")
+	}
+	err = tkgClient.TKGConfigReaderWriter().Init(tkgConfigFile)
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to initialize the TKG config reader writer")
+	}
```

Comment:
might be good to add a comment to describe why we are doing this as it is not clear from the function invocation.